README.md

```diff
@@ -42,43 +90,42 @@ cd build
-
-#### Search for project by name  
-![ Search for project by name](https://raw.githubusercontent.com/alex1701c/Screenshots/master/JetBrainsRunner/launch_by_name.png)
+#### 2.2 Restart KRunner
+```
+kquitapp5 krunner;kstart5 krunner
+```
 
-#### Search projects of app  
-![ Search for project by name](https://raw.githubusercontent.com/alex1701c/Screenshots/master/JetBrainsRunner/search_projects_of_app.png)
-
-#### Config Dialog  
-The first checkbox enables search suggestions based on the application name (like in the first and third screenshot).  
-The second checkbox enables suggestions based on the project name, this is shown in the second screenshot.  
-The last checkbox enables notifications you when a new version of this plugin has been released.
-It checks for a new version weekly or when the config is opened. By unchecking it you turn the check permanently off.  
-After this you can change the displayed text for the run options using the displayed rules.  
-If the plugin has issues finding the installed applications or finding the correct config file you can specify them manually.
-Once the files have been selected the icon, name and recent project get displayed.  
-If you have to do manual you are welcome to create a pull request and add the application path to the default configuration:
-https://github.com/alex1701c/JetBrainsRunner/blob/master/src/JetbrainsApplication.cpp#L137
-![Config Dialog](https://raw.githubusercontent.com/alex1701c/Screenshots/master/JetBrainsRunner/config_dialog.png)
+#### 2.3 (Optional) Configure Plugin
+1. Go to system settings (`systemsettings5`)
+2. Search for `krunner`
+3. Search for `JetBrains` in KRunner plugin list
+4. Configure plugin
+
+#### Config Dialog
+#### Checkbox Options
+1. Search By Application: Enables search suggestions based on the Application name (like in the [first](#available-clion-projects) and [third](#search-projects-of-app) screenshot).  
+2. Search By Project: Enables suggestions based on the Project name, this is shown in the [second](#search-for-project-by-name) screenshot.  
+3. Update Notifications: Enables notifications you when a new version of this plugin has been released. It checks for a new version weekly or when the config is opened. By unchecking it you turn the check permanently off.
+4. Show use application name as catagory for projects.
+
+#### Result Formatting
+You can change the displayed text for the run options using the following rules.
+- *%APPNAME*: application name
+- *%APP*: shortened application name
+- *%PROJECT*: project name
+- *%DIR*: full directory path
+
+#### Manually Adding IDE's
+If the plugin has issues finding the installed applications or finding the correct config file you can specify them manually, you are welcome to create a pull request and add it to the [default configuration](https://github.com/alex1701c/JetBrainsRunner/blob/master/src/JetbrainsApplication.cpp#L137).
```

Comment:
I realized that this is out of sync with a change I did some time ago.

Now the file paths are no longer hardcoded. If one encounters problems that require manual configuration, it is be best to create an issue here on Github.